modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -1089,12 +1090,13 @@ export default class I3SConverter {
 
     const options = await this._fetchPreloadOptions();
     this.sourceTileset.options = {...this.sourceTileset.options, ...options};
+
     if (options.headers) {
-      this.sourceTileset.fetchOptions.headers = options.headers;
+      this.sourceTileset.loadOptions.fetch.headers = options.headers;
       console.log('Authorization Bearer token has been updated'); // eslint-disable-line no-undef, no-console
     }
     if (options.token) {
-      this.sourceTileset.fetchOptions.token = options.token;
+      this.sourceTileset.loadOptions.fetch.token = options.token;
```

Comment:
loadOptions.fetch should be a [`RequestInit`]*https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) object accepted by fetch. 

I don't think is has `token`.